app/src/main/java/com/project/gamedb/ui/main/MainInterface.kt

```diff
@@ -0,0 +1,7 @@
+package com.project.gamedb.ui.main
+
+import androidx.fragment.app.Fragment
+
+interface MainInterface {
```

Comment:
Đặt tên khác miêu tả đúng chức năng của nó đi anh 
```suggestion
interface OnFragmentIntegrationListener {
```